### PR TITLE
Color Audit Pt 1

### PR DESCRIPTION
**Pull request by gettinToasty**
Implementation of scoped variables for all of our colors. Dark mode is not yet working in the twitch tags input but apart from some slight intentional differences in the name of color unification this is going well so far

### CODE REVIEW

File: app/app.g.less [hunk 1]
```diff
@@ -327,14 +327,14 @@ select {
...
+  background: var(--section);
   border-radius: 0 3px 3px 0;
 }
 
 ::-webkit-scrollbar-thumb {
-  background-color: @white;
-  border: 2px solid @day-border;
-  border-right: 1px solid @day-border;
+  background-color: var(--scrollbar);
+  border: 2px solid var(--border);
```
Comment (blackxored) [inline app/app.g.less]: I think this one should be called `border-style`

File: app/app.g.less [hunk 2]
```diff
@@ -368,7 +368,8 @@ select {
 }
 
 .v--modal {
-  background-color: @day-bg;
+  background-color: var(--background);
+  box-shadow: 0 0 0 var(--shadow);
```
Comment (blackxored) [inline app/app.g.less]: Does this do anything? Is it animated later?

Comment (gettinToasty) [reply app/app.g.less]: not sure, just trying to change it the least from staging

File: app/app.g.less (hunk 1 above)
Comment (gettinToasty) [reply app/app.g.less]: can you explain your thoughts here? why border and not suffixing any of the other colors?

**Pull request merged**